org.cubocore.CoreGarage.yml

```diff
@@ -0,0 +1,55 @@
+app-id: org.cubocore.CoreGarage
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: coregarage
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home   # Required to archive any files and open any archived files
+  - --filesystem=xdg-config:create
```

Comment:
why do you need this (line 11)

Reply:
we need xdg-config permission but if we use xdg-config the flatpak-builder-linter causes an error as finish-args-arbitrary-xdg-config-access but using the :create permission it works since we have to read and write the app configuration in the ~/.config directory.

Reply:
I don't understand why you need this permission.

And thanks for the linter bug report. It will be fixed.

Reply:
Because in our app suite we store everything in the config folders, like recent file history, settings, bookmarks. and all of this needs to be accessed by the all the apps from same file, so having access to config is essential. also we don't have setting for each app separately, coregarage is our settings app, it modify the settings file and all the apps read from the file, so common file is necessary.

Reply:
and please don't hold this PR for the bug fix.

Reply:
> and please don't hold this PR for the flatpak bug fix.

The linter is getting a fix.

But then the domain or lack of control thereof is blocking.

Reply:
we don't have money to renew the domain immediately, so hope you guys will understand the circumstance.